test/powershell/Modules/Microsoft.PowerShell.Utility/JsonObject.Tests.ps1

```diff
@@ -1,46 +1,47 @@
 ﻿Describe 'Unit tests for JsonObject' -tags "CI" {
 
-    function ShouldThrow
-    {
-        param (
-            [Parameter(ValueFromPipeline = $true)]
-            $InputObject,
-            [Parameter(Position = 0)]
-            $ExpectedException
-        )
-
-        try
-        {
-            & $InputObject
-            throw "Should throw exception"
-        }
-        catch
-        {
-            $_.FullyQualifiedErrorId | should be $ExpectedException
-        }
-    }
-
     $validStrings = @(
-        @{ name = "empty"; str = "" }
-        @{ name = "spaces"; str = "  " }
-        @{ name = "object"; str = "{a:1}" }
+        @{ name = "null";   str = $null;   ReturnHashTable = $true  }
+        @{ name = "empty";  str = "";      ReturnHashTable = $true  }
+        @{ name = "spaces"; str = "  ";    ReturnHashTable = $true  }
+        @{ name = "object"; str = "{a:1}"; ReturnHashTable = $true  }
+        @{ name = "null";   str = $null;   ReturnHashTable = $false }
+        @{ name = "empty";  str = "";      ReturnHashTable = $false }
+        @{ name = "spaces"; str = "  ";    ReturnHashTable = $false }
+        @{ name = "object"; str = "{a:1}"; ReturnHashTable = $false }
     )
 
-    It 'no error for valid string - <name>' -TestCase $validStrings {
-        param ($str)
+    It 'no error for valid string ''<name>'' with -ReturnHashTable:$<ReturnHashTable>' -TestCase $validStrings {
```

Comment:
Please start test descriptions with capital.

We use `$validStrings` one time. Please use our common template:
```powershell
It 'no error for valid string ''<name>'' with -ReturnHashTable:$<ReturnHashTable>' -TestCase @(
 @{ name = "null";   str = $null;   ReturnHashTable = $true  }
 ...
) {
...
}
```